setup.py

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env python
 
+# Copyright (C) 2008-2016 The scikit-sparse developers
```

Comment:
I think this should go to the bottom of the list and only be a (C) since 2016 (but IANAL).